src/styles/main.scss

```diff
@@ -0,0 +1,57 @@
+@import './init/variables';
+@import './blocks/page';
+
+.calendar {
+  flex-wrap: wrap;
+  gap: 1px;
+  padding: 10px;
+  display: flex;
+  width: ($widthDays + $gapDays) * 7;
```

Comment:
The calculation for the calendar width `($widthDays + $gapDays) * 7` assumes that the gap is included in the width of each day. Ensure that this calculation aligns with your design expectations, as the gap might need to be subtracted if it's not part of the width.